phoenix-core/src/main/java/org/apache/phoenix/util/DateUtil.java

```diff
@@ -112,10 +112,12 @@ public static TimeZone getTimeZone(String timeZoneId) {
         int maxOrdinal = Integer.MIN_VALUE;
         List<PDataType> timeDataTypes = Lists.newArrayListWithExpectedSize(6);
         for (PDataType type : PDataType.values()) {
-            if (java.util.Date.class.isAssignableFrom(type.getJavaClass())) {
-                timeDataTypes.add(type);
-                if (type.ordinal() > maxOrdinal) {
-                    maxOrdinal = type.ordinal();
+            if(type != null) {
```

Comment:
what's this for?

Reply:
For the 70 hack, as we had a bunch of nulls in there.  Can remove with your 48 ordinal.

Reply:
Yup, can be removed.